packages/ui-file-drop/src/FileDrop/index.tsx

```diff
@@ -76,20 +64,23 @@ class FileDrop extends Component<FileDropProps, FileDropState> {
   static propTypes = propTypes
   static allowedProps = allowedProps
   static defaultProps = {
-    // @ts-expect-error ts-migrate(6133) FIXME: 'e' is declared but its value is never read.
-    onClick: function (e: any) {},
-    // @ts-expect-error ts-migrate(6133) FIXME: 'accepted' is declared but its value is never read... Remove this comment to see the full error message
-    onDrop: function (accepted: any, rejected: any, e: any) {},
-    // @ts-expect-error ts-migrate(6133) FIXME: 'accepted' is declared but its value is never read... Remove this comment to see the full error message
-    onDropAccepted: function (accepted: any, e: any) {},
-    // @ts-expect-error ts-migrate(6133) FIXME: 'rejected' is declared but its value is never read... Remove this comment to see the full error message
-    onDropRejected: function (rejected: any, e: any) {},
-    // @ts-expect-error ts-migrate(6133) FIXME: 'e' is declared but its value is never read.
-    onDragEnter: function (e: any) {},
-    // @ts-expect-error ts-migrate(6133) FIXME: 'e' is declared but its value is never read.
-    onDragOver: function (e: any) {},
-    // @ts-expect-error ts-migrate(6133) FIXME: 'e' is declared but its value is never read.
-    onDragLeave: function (e: any) {},
+    onClick: function (_e: React.MouseEvent) {},
+    onDrop: function (
+      _accepted: ArrayLike<DataTransferItem | File>,
+      _rejected: ArrayLike<DataTransferItem | File>,
+      _e: React.DragEvent
+    ) {},
+    onDropAccepted: function (
+      _accepted: ArrayLike<DataTransferItem | File>,
+      _e: React.DragEvent
+    ) {},
+    onDropRejected: function (
+      _rejected: ArrayLike<DataTransferItem | File>,
+      _e: React.DragEvent
+    ) {},
+    onDragEnter: function (_e: React.DragEvent) {},
+    onDragOver: function (_e: React.DragEvent) {},
+    onDragLeave: function (_e: React.DragEvent) {},
```

Comment:
I think we settled on not doing this with event handler callback functions, so you can delete these from the defaultProps.